pixart.js

```diff
@@ -1 +1,37 @@
+$(document).ready(function(){
+  $(".brush").css("background-color","red");
+});
```

Comment:
everything should be wrapped inside the document.ready function as this ensures all of our code is ready upon page load.